src/slack/tests/main_test.bal

```diff
@@ -82,23 +82,23 @@ function deleteMessageAfterTest() {
 function testListConversations() {
     var response = convClient->listConversations();
     if (response is error) {
-        test:assertFail(msg = <string>response.detail()?.message);
+        test:assertFail(msg = <string>response.detail()["message"]);
```

Comment:
```suggestion
        test:assertFail(msg = <string>response.message());
```

Reply:
```suggestion
        test:assertFail(msg = response.detail()["message"]);
```
Shall we remove `<string>` casting as well?

Reply:
Done